src/main/java/com/nutomic/syncthingandroid/fragments/SettingsFragment.java

```diff
@@ -297,6 +318,22 @@ public boolean onPreferenceChange(Preference preference, Object o) {
         return true;
     }
 
+    @Nullable
+    private String formatWifiNameList(Set<String> ssids) {
+        if (ssids != null) {
+            ssids = new TreeSet<>(ssids);
+            StringBuilder sb = new StringBuilder();
+            for (String ssid : ssids) {
+                if (sb.length() > 0) {
+                    sb.append(", ");
+                }
+                sb.append(ssid.replaceFirst("^\"", "").replaceFirst("\"$", ""));
+            }
```

Comment:
Since you are the owner of this project I'll follow along. But I get the feeling you are bit nitpicking about everything you did not code yourself.